plugins/com.google.cloud.tools.eclipse.dataflow.ui.test/src/com/google/cloud/tools/eclipse/dataflow/ui/page/MessagesTest.java

```diff
@@ -19,6 +19,8 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import com.google.cloud.tools.eclipse.dataflow.ui.Messages;
```

Comment:
Why don't you move the tests in this class to `.dataflow.ui.Messages` you just added here, and then delete this class?